### PR TITLE
v0.72.0 - popOver tweaks

### PR DESCRIPTION
**Pull request by sonnyprince**
_PopOver Tweaks._

- Change of popover JS class
- Addition of content section within popover so long content can scroll
- Bump of versions

popover-content

## UI Review Checks

- [x] This PR has been checked with regard to our brand guidelines
- [x] UI Documentation has been [created|updated]
- [x] This code has been checked with regard to our accessibility standards

## Browsers Tested
- [x] Mobile

### CODE REVIEW

File: src/scss/components/_fullscreen-pop-over.scss [hunk 1]
```diff
@@ -78,3 +79,11 @@ $fullScreenPopOver-shadow-color      : rgba($black, 0.12);
         bottom: 0;
         box-shadow: 0 -2px 2px 0 $grey--lighter;
     }
+
+    .c-fullScreenPopOver-content {
+        height: 100%;
+        overflow-x: hidden;
```
[the remark this answers is not in the document]
Comment (sonnyprince) [reply src/scss/components/_fullscreen-pop-over.scss]: adding the `overflow-x hidden` should hide a horizontal scrollbar :)

[the remark this answers is not in the document]
Comment (ashleynolan) [reply src/scss/components/_fullscreen-pop-over.scss]: Does that appear then by adding `overflow-y` scroll?

[the remark this answers is not in the document]
Comment (sonnyprince) [reply src/scss/components/_fullscreen-pop-over.scss]: not from my tests. just in case i guess 😄 
would you like me to remove?

**Comment by sonnyprince** [comment]
bumped version down to 0.72.0. @DamianMullins was happy for me to do so :)

**Comment by sonnyprince** [comment]
@oanuna could you ✅from changes you requested :)

**Pull request merged**